### ISSUE TITLE
Smart health related checks are executed against hardware raids

### ISSUE THREAD

**Issue by err404r** [canonical/hardware-observer-operator#357]
New revision of Hardware observer introduced checks for smart drives, unfortunately collector is trying to run checks against all block devices including hardware raids. They are failing and generating false positive alerts.

**Comment by aieri** [comment]
@err404r yes, the exporter doesn't really distinguish between logical and physical block devices and exports metrics for anything listed by `smartctl --scan`. Which alerts are firing? Which metrics are triggering the alerts?

**Comment by err404r** [comment]
Alert name `SmartHealthStatusFail` 
Rule:
`smartctl_device_smart_status{juju_application="hardware-observer",juju_model="ceph",juju_model_uuid="bdb2f523-0ca5-463b-8061-8686da1151db"} == 0`

**Comment by Deezzir** [comment]
@err404r Can you confirm that the alerts which are firing miss the device label?
